.travis.yml

```diff
@@ -28,9 +28,10 @@ addons:
       - valgrind
       - voms-dev
       - gdb
+      - libarchive-dev
 
 before_script:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew cask install osxfuse; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew cask install osxfuse && brew install libarchive; fi
```

Comment:
So... on travis macOS nodes libarchive is missing? Isn't that a default library that comes with macOS?